.travis.yml

```diff
@@ -19,6 +19,10 @@ matrix:
       env:
         - OS_TYPE=doxygen
       addons: { apt: { packages: ["cmake", "graphviz"] } }
+    - os: linux
+      env:
+        - OS_TYPE=clangformat
+      addons: { apt: { packages: ["clang-format-6.0"], sources: ["llvm-toolchain-xenial-6.0"] } }
```

Comment:
curios: would this work?  `addons: { apt: { packages: ["clang-format-6.0"], sources: ["llvm-toolchain-${TRAVIS_DIST}-6.0"] } }`

also, is ther a generic `llvm-toolchain-6.0` addon?
i would not add `xenial` if possible; otherwise, it's probably better to add `dist: xenial` back to make sure things stay consistent (sorry about that, if it happens)